app/controllers/computacenter/multi_domain_chromebooks_iss_scl_controller.rb

```diff
@@ -0,0 +1,44 @@
+require 'csv'
+
+class Computacenter::MultiDomainChromebooksIssSclController < Computacenter::BaseController
+  def index
+    @schools = School
+    .includes(:responsible_body)
+    .joins(:preorder_information)
+    .gias_status_open
+    .la_funded_provision
+    .where({
+      provision_type: %w[iss scl],
+      preorder_information: {
+        will_need_chromebooks: %w[yes i_dont_know],
+      },
+    })
+    .sort_by { |s| s.responsible_body.name }
```

Comment:
Sorting via SQL is a little messy with includes in the query it generates, for the number of rows expected this seems cleaner.